REVIEWING.md

```diff
@@ -0,0 +1,108 @@
+
+Periods when it is acceptable to merge breaking changes will be widely announced.
+If it is not one of those periods, reviewers should be on the lookout for breaking changes.
+PRs with breaking changes should not be merged.
+Please guide the contributor in how to make the change in a non-breaking fashion.
+If this is not possible, deferring the PR to the next breaking change window or closing it entirely is appropriate.
+
+### Commit Messages and Hygiene
+
+Good commit messages are essential for understanding why a change was made in the future.
+Reviewers should check each commit of a PR to ensure that it has an appropriate commit message which fully and accurately explains the change and why it is being made.
+This should remain true after changes to the PR due to code review, so please re-review commit messages before merging code, to ensure they are still accurate.
+
+Each commit in a PR should be self-contained and have a clear and distinct purpose.
+If this is not true, please encourage the contributor to squash their commits using `git rebase -i` until it is true.
+
+### Disagreements between reviewers
+
+We do not expect all reviewers to be of the same opinion during code review.
+If you see another reviewer requesting changes to a PR you do not agree with, it is perfectly acceptable to comment to that effect.
+Disagreements between maintainers can generally be worked out during PR review through comments.
+If this is not possible, other maintainers can be called on to give their opinions and determine a way forward.
+We encourage such disagreements and discussions, so long as they remain respectful.
+
+Being respectful includes respect for the contributor's time.
+If there is a disagreement, please do not make the contributor make repeated changes until an agreement on how to proceed is reached.
+Also, please attempt to reach an agreement quickly, so the PR can be merged in a timely fashion.
+
+### Language Version Updates
+
+Most repositories in the Podman project are written in Go, and as such target a specific version of Go in their `go.mod`.
+For example:
+```
+$ cat go.mod | grep 'go 1.'
+go 1.22.8
+```
+
+For all branches except the main branch, this should always remain static.
+PRs into a non-main branch which change the supported Go version should be modified to not require such a change, or rejected if that is not possible.
+Changing supported Go version in the main branch is allowed, but not encouraged.
+Doing so will require significant discussion among maintainers, as it can affect what distributions the main branch can be built.
+
+### Tests
+
+Please Check tests added by the PR.
+If a PR has no new tests, determine if this is actually appropriate - has new functionality been added which should have been tested?
+If the PR does have tests, check to see if they are reasonably comprehensive.
+The Podman Project does not have code coverage standards at present, but we aim to ensure that all new functionality is tested.
```

Comment:
Also make sure the newly added test for a bug fix/feature fails against a binary from current main. If that is not the case the test is not good as it does not check the newly added change. As such it should be modified until that this is the case.